Sources/Tests/Extraction/Filters/Test_OrientedSmoother.c

```diff
@@ -146,4 +157,20 @@ TEST(OrientedSmoother, VisualiseSmoother)
   BinaryMapToImage(&binarized, &newV);
 
   pgm_write("../TestImages/Person1/output-binarised-Hamster-1.0.pgm", &newV);
+

```

Comment:
It could just be doing this on github, but again the tab depth here looks like 2, rather than 4...